### PR TITLE
Update Lua file name

### PR DESCRIPTION
**Pull request by JyeSmith**
This is a draft and I have no preference if the name is changed to elrsV3 or something more generic.  Or even pull lua out into its own repo?

But it should be changed before the next RC to minimise user confusion.

Thoughts?

### CODE REVIEW

**Comment by pkendall64** [comment]
If it's not called `elrsV3.lua` (or similar) then configurator is not going to pick it up.
Configurator has a regexp to detect the files and the pattern is `elrs.*\.lua` which is then sorted in reverse order and the first item is picked. So that means the latest version in the repository checkout is the version that is downloaded.

**Comment by JyeSmith** [comment]
Ive updated the file to elrsV3.  Lets go with this for the V3 release and then think about a long term solution.

File: src/lua/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 # Which file do I need?
-* For transmitter on master / ExpressLRS 2.0 -> elrsV2.lua
+* For transmitter on master / ExpressLRS 3.0 -> elrsV3.lua
```
Comment (CapnBry) [inline src/lua/README.md:3]: For transmitter on master, ExpressLRS 2.x, or 3.0?

**Review by CapnBry — APPROVED** [review]

**Review by pkendall64 — APPROVED** [review]

**Review by StonedDawg — APPROVED** [review]

**Comment by pfeerick** [comment]
Could always rename `elrs.lua` to `elrsV1.lua`, and then link `elrs.lua` to the latest lua script (i.e. elrsV3.lua atm), so 'master' will always be latest. I have ELRS, ELRSv2 and ELRSv3 on my SD card (with amended `TNS|ExpressLRS|TNE` tags so each shows it's version) so all bases are covered 🙃 😆

**Comment by CapnBry** [comment]
> I have ELRS, ELRSv2 and ELRSv3 on my SD card (with amended `TNS|ExpressLRS|TNE` tags so each shows it's version) so all bases are covered 🙃 😆

You don't need 3x ELRS.luas on your SD card. There's just two versions, the original, for v1.x, and the one that works with everything 2.x on. Your naming makes more sense, where elrsv1.lua was for 1.x and elrs.lua works with everything beyond that, but I think having the current named the same as the original is a support nightmare. We definitely need to know if the user is running the V2 script (which the "elrsv3.lua" is, it is V2 of the protocl, just renamed) for when they say "I launch the Lua and it doesn't load".  Users don't even know if they downloaded the lua file or an HTML github page, there's zero chance they're going to know if the have the original elrs.lua vs the current elrs.lua. :(

**Comment by pfeerick** [comment]
> Users don't even know if they downloaded the lua file or an HTML github page, 

This is why support is already a nightmare... 🙃 So yeah, ok, not such a good idea to make it worse. I do understand the Lua for is the same for V2/V3, so will probably get around to changing the display name to `ExpressLRS 2/3` on my local copy "at some point"... when I initially added it I just wanted to make sure as nothing worse than needing the other version and not having it when out at the field... 😉

**Comment by CapnBry** [comment]
Truth. "V2" was the config protocol version, and it was a poor choice considering it was coincidentally 2.x of ExpressLRS. Technically it was elrs-crsf.lua, since "V2" is standard CRSF config protocol. All the TBS utilities still work against 2.x or 3.x too.

The lua we're releasing for 3.x is slightly different than the 2.x version, because when you change a value it now automatically refreshes all same-level items, so like when you select Full Res vs Std Res modes it changes the switch mode options too. In the 2.x script you'd refresh manually for the change to show up. So both still work, but the 3.x version has a better user experience on 3.x and works the same on 2.x. 

We're going to have to split the Lua up to have better EdgeTX widget support and launch right from the main screen, so this will all go away at some point because there will be more files and not just one confusingly-named thing.

**Comment by pfeerick** [comment]
Oh no you didn't just say that... 

> this will all go away at some point because there will be more files

.. ah hem... were *all* files downloaded and put in the same folder? (*no comment*) 😇 

And yes, that was the really nice thing with V2... that Agent86 or whatever Lua app works with it... or you can use the ELRS lua instead of it.  :grin: Now, if only SPI V2 and BF would work... 😆 I'm going to have to leave a radio on ELRS v2 for now as I have (thankfully only one) a mini quad with SPI receiver I fly a lot - the others I added EP2s to so they're fine. And at least the two SPRacingH7RF FCs I got have enough UARTS to ignore the onboard ELRS... SPI Rx is a nice idea, but so impractical if ELRS continues to update with new cool stuff... you guys needs to slow down so BF can catch up! 🤣

**Comment by CapnBry** [comment]
> .. ah hem... were _all_ files downloaded and put in the same folder? (_no comment_)

I didn't say that. It will be a proper collection of Lua scripts like the GUI TBS Agent or BF Lua etc.

SPI RX will have an 3.x compatibility update PR up later today with binaries. It will not move forward after this, and does not have full 3.0 support.

**Pull request merged**